infrastructure/modules/monitoring/fluent-bit/configs/input/input-kubernetes-daemonset.conf

```diff
@@ -1,8 +1,21 @@
 [INPUT]
     Name               tail
     Tag                kube.*
-    Path               /var/log/containers/control-plane*.log, /var/log/containers/compute-plane*.log, /var/log/containers/armonik-metrics-exporter*.log, /var/log/containers/ingress*.log, /var/log/containers/keda-operator*.log, /var/log/containers/mongodb*.log
```

Comment:
I fail to see why the conf has to split in 2, and more importantly, why the list of logs to watch is different.
Could you explain a bit?

Reply:
The tag kube.* is for Seq and the tag application.* is for cloudwatch

Reply:
I am not talking about the `Tag`, but about the `Path`

Reply:
Excerpt of offline discussion:

> We don't want the same logs on cloudwatch than with seq.

Reply:
We do not need logs of MongoDB, NGINX and Keda in Seq